### DISCUSSION TITLE
what it is? - Conventions

### DISCUSSION THREAD

**Discussion by Arexils** [squidfunk/mkdocs-material#6045] [Q&A] [answered]
![image](https://github.com/squidfunk/mkdocs-material/assets/20801433/0fc4cb5c-46ca-4e70-a5d6-a440670a3716)
What is it called and how do you do such a thing?
Where can this be reviewed?

**Comment by alexvoss** [comment, marked as answer]
Hi,

do you mean specifically the circled bit?

Generally, the availability annotations for features are comments in the page source that are converted in [.overrides/hooks/shortcodes.py](https://github.com/squidfunk/mkdocs-material/blob/master/src/.overrides/hooks/shortcodes.py). This is code specific to the mkdocs-material documentation, so you would need to write your own equivalent code.

Cheers, Alex

**Reply by squidfunk** [reply]
Correct! Also, if you click on the icon, it directs you to the [conventions](https://squidfunk.github.io/mkdocs-material/conventions/) page where the meaning of each icon is explained.

**Comment by Arexils** [comment]
Got it, thanks!)
